pyClient/zeth/joinsplit.py

```diff
@@ -292,64 +270,34 @@ def encode_pub_input_to_hash(message_list: List[Union[str, List[str]]]) -> bytes
     return input_sha
 
 
-def field_elements_to_hex(longfield: str, shortfield: str) -> str:
-    """
-    Encode a 256 bit array written over two field elements into a single 32
-    byte long hex
-
-    if A= x0 ... x255 and B = y0 ... y7, returns R = hex(x255 ... x3 || y7 y6 y5)
-    """
-    # Convert longfield into a 253 bit long array
-    long_bit = "{0:b}".format(int(longfield, 16))
-    if len(long_bit) > 253:
-        long_bit = long_bit[:253]
-    long_bit = "0"*(253-len(long_bit)) + long_bit
-
-    # Convert shortfield into a 3 bit long array
-    short_bit = "{0:b}".format(int(shortfield, 16))
-    if len(short_bit) < 3:
-        short_bit = "0"*(3-len(short_bit)) + short_bit
-
-    # Reverse the bit arrays
-    reversed_long = long_bit[::-1]
-    reversed_short = short_bit[::-1]
-
-    # Fill the result 256 bit long array
-    res = reversed_long[:253]
-    res += reversed_short[:3]
-    res = hex_extend_32bytes("{0:0>4X}".format(int(res, 2)))
-
-    return res
-
-
-def sign(
-        keypair: JoinsplitKeypair,
-        hash_ciphers: str,
-        hash_proof: str,
-        hash_inputs: str) -> int:
+def _sign(
```

Comment:
This needs to be moved to `signing.py` which should contain all the algorithms of the signature scheme we use. Namely, `Sign(sk, msg)` (returns `sigma`), `Verify(vk, sigma, msg)` (return 0/1), `KeyGen(\secparam)` (returns `sk, vk`).

Thus, the signature of `_sign` above function needs to be modified (cc: @rrtoledo as this is related to the thread:https://github.com/clearmatics/zeth/pull/122#discussion_r356465339)